kaizen/cli.py

```
@@ -1,64 +1,108 @@
+        results = runner.run_tests(resolved_file_path)
         
         # Check if any tests failed
         failed_tests = []
         for region, result in results.items():
-            for test_case in result.get('test_cases', []):
-                if test_case['status'] != 'passed':
+            if not isinstance(result, dict):
+                click.echo(f"Warning: Invalid result format for region {region}: {result}")
+                continue
+                
+            test_cases = result.get('test_cases', [])
```

Pylint (3.3.7)

[error] Instance of 'str' has no 'get' member

(E1101)